backends/cxxrtl/runtime/cxxrtl/cxxrtl.h

```diff
@@ -1582,7 +1582,7 @@ struct module {
 
 	// Compatibility method.
 #if __has_attribute(deprecated)
-	__attribute__((deprecated("Use `debug_info(&items, /*scopes=*/nullptr, path);` instead. (`path` could be \"top \".)")))
```

Comment:
This particular line was previously fixed in https://github.com/YosysHQ/yosys/pull/4432/files. (I don't really have an opinion on which message is better; this is a comment on the merge conflict.)

Reply:
Yepyep — I resolved the conflict by preferring mine (as they modified the base of the deprecation message the same way), since the `` `path` could be "top ". `` bit seems confusing at best.

(At least, the first time I saw this message, it seemed to imply I'd need to add some kind of path, otherwise why suggest a value for one? But callers of the deprecated API already have a `path` value, since the signature is indeed `void debug_info(debug_items &items, std::string path)`, so it seems best to just remove the suggestion.)

Reply:
Sounds good.